appveyor.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+install:
+# TODO: Switch back to chocolatey once 1.12.0 is available.
```

Comment:
Note: this won't be required once 1.12.0 is stable (and available via chocolatey). You might want to open an issue to track this.